pubspec.yaml

```diff
@@ -1,10 +1,13 @@
 name: json_diff
-version: 0.1.2
+version: 0.2.0
```

Comment:
I don't think this requires a bump to 0.2.0. It's ok to bump dependency versions in a minor bump. WDYT about 0.1.3?

Reply:
sure why not.